src/start.js

```diff
@@ -32,11 +32,11 @@ export const start = command('start', {
   let { default: { spawn } } = yield import('cross-spawn');
   let { host, port } = flags;
 
-  let args = ['--ci', `--host=${host}`, `--port=${port}`, ...argv];
-  log.info(`Running "start-storybook ${args.join(' ')}"`);
+  let args = ['dev', '--ci', `--host=${host}`, `--port=${port}`, ...argv];
```

Comment:
we'll need backward compatibility with v6.

Reply:
Yep, this is a little tricky.
I am unsure how to structure the SDK in that case.